lark/parser_frontends.py

```diff
@@ -71,6 +77,18 @@ def _get_lexer_callbacks(transformer, terminals):
             result[terminal.name] = callback
     return result
 
+class PostLexConnector:
+    def __init__(self, lexer, postlexer):
+        self.lexer = lexer
+        self.postlexer = postlexer
+
+    def make_lexer_state(self, text):
+        return self.lexer.make_lexer_state(text)
+
+    def lex(self, lexer_state, parser_state):
+        i = self.lexer.lex(lexer_state, parser_state)
+        return self.postlexer.process(i)
```

Comment:
Could it be interesting for the postlexer to access the lexer_state? I currently can not imagine a situation that would be solved by a CustomLexer anyway.

Reply:
It might be interesting, though I can't think of a specific case either. But I plan to expose it in v1.0

Reply:
Yeah, let it be. Not worth the hassle if we can't think of a usecase.